src/scss/_statements.scss

```diff
@@ -24,184 +24,176 @@
+  (color, $drops-heading--color),
+
+  (display, block),
+  (margin, 0),
+  (white-space, nowrap),
+));
 
 
 /**
  * anchor
  */
-$drop-anchor--selector: selector--set-element($drop--selector, 'anchor');
-
-@include selector($drop-anchor--selector) {
-  @include declaration('padding-top', $drops-anchor--padding-y);
-  @include declaration('padding-bottom', $drops-anchor--padding-y);
-  @include declaration('padding-left', $drops--padding-x);
-  @include declaration('padding-right', $drops--padding-x);
-  @include declaration('font-size', $drops-anchor--font-size);
-  @include declaration('line-height', $drops-anchor--line-height);
-  @include declaration('letter-spacing', $drops-anchor--letter-spacing);
-  @include declaration('font-weight', $drops-anchor--font-weight);
-  @include declaration('text-transform', $drops-anchor--text-transform);
-  @include declaration('text-decoration', $drops-anchor--text-decoration);
-  @include declaration('color', $drops-anchor--color);
-  @include declaration('background-color', $drops-anchor--background-color);
-
-  @include declaration('display', block);
-  @include declaration('width', 100%, ('<button'));
-  @include declaration('text-align', inherit, ('<button'));
-  @include declaration('clear', both);
-  @include declaration('background', none, ('<button'));
-  @include declaration('border', 0, ('<button'));
-  @include declaration('white-space', no-wrap);
-  @include block-generation($drop-anchor--selector);
-}
+$drop-anchor--selector: selector(set $drop--selector element 'anchor');
+
+@include statement($drop-anchor--selector, (
+  (padding-top, $drops-anchor--padding-y),
+  (padding-bottom, $drops-anchor--padding-y),
+  (padding-left, $drops--padding-x),
+  (padding-right, $drops--padding-x),
+  (font-size, $drops-anchor--font-size),
+  (line-height, $drops-anchor--line-height),
+  (letter-spacing, $drops-anchor--letter-spacing),
+  (font-weight, $drops-anchor--font-weight),
+  (text-transform, $drops-anchor--text-transform),
+  (text-decoration, $drops-anchor--text-decoration),
+  (color, $drops-anchor--color),
+  (background-color, $drops-anchor--background-color),
+
+  (display, block),
+  (width, 100%, ('<button')),
+  (text-align, inherit, ('<button')),
+  (clear, both),
+  (background, none, ('<button')),
+  (border, 0, ('<button')),
+  (white-space, no-wrap),
+));
 
 
 /**
  * divider
  */
-$drop-divider--selector: selector--set-element($drop--selector, 'divider');
+$drop-divider--selector: selector(set $drop--selector element 'divider');
 
-@include selector($drop-divider--selector) {
-  @include declaration('-include-drops-divider--border-width', $drops--border-width);
-  @include declaration('border-color', $drops--border-color);
-  @include declaration('margin', $drops-divider--margin);
-  @include block-generation($drop-divider--selector);
-}
+@include statement($drop-divider--selector, (
+  ('-include-drops-divider--border-width', $drops--border-width),
```

Comment:
the syntax for mixins has changed with v4. It should be written in the way below

```scss
('@include drops-divider--border-width', $drops--border-width),
```